section3/reflection.md

```diff
@@ -1,17 +1,19 @@
 ## Section 3 Reflection
 
 1. How different did your workflow feel this week, considering we asked you to follow the Pomodoro technique?
-
+* I didn't particularly enjoy it because it feels like to many brakes. I understand that brakes are necessary to create good work but to me I feel like I do better if I dive into something for about 50 minutes and then maybe take a break. If I'm in a good workflow I like to keep going until the job is done. This behavior come from being a mechanic in the military, where the day is not over until the job is done most of the times. It was common to work through lunches and physical training to get the job done.
```

Comment:
If 50 minutes on, then a break, works for you - I recommend using those time frames for when you work solo and/or suggesting it when you work in teams!! 25/5 doesn't work for everyone.